docs/source/external-tools/cdf/cdf_requirements.rst

```diff
@@ -71,7 +151,9 @@ This attribute determines the file naming convention and is used by CDA Web.  It
 
 * Source_name - (e.g. ``imap``)
 * Descriptor - (e.g. the instrument, see above)
-* Data_type - (e.g. the mode, data level, and descriptor)
+* Data_type - (e.g. data level, and descriptor)
+
+These three attributes can be ordered in different ways based on the user's filename convention. SPDF uses default naming conventions of ``source_datatype_descriptor_yyyyMMdd`` but IMAP uses this convention, ``source_descriptor_datatype_yyyyMMdd_vNNN``.
```

Comment:
I wonder if you could add an example which shows the format the instrument (descriptor) takes in the string, because I could imagine it being the shorter name (e.g. 'MAG') the longer name ('Magnetometer') or the full instrument name above ('MAG>Magnetometer').

Reply:
Descriptor is already described in the naming convention doc, so I'd prefer to link to that document rather than rewriting it here

Reply:
I did think about including example but didn't know if I should. I was afraid it may cause confusion. If we think it will help, I am happy to include it.

For example, `source_datatype_descriptor_yyyyMMdd` would result in filename convention like this `imap_l1a_norm-raw_mag_20241122` whereas `source_descriptor_datatype_yyyyMMdd_vNNN` would result in this filename convention `imap_mag_l1a_norm-raw_20241122_v001`. If you guys can suggested a good wording to say that, that may be helpful.

Reply:
@maxinelasp A link is fine too!

I suppose I had figured anyone referring to the documentation here would want to use the IMAP convention, but maybe that's not a safe assumption.

Reply:
A quick summary and example may be helpful still. I added both. I will fix the format but let me know what you both think?
```suggestion
These three attributes can be ordered in different ways based on the user's filename convention. SPDF uses default naming conventions of ``source_datatype_descriptor_yyyyMMdd`` but IMAP uses this convention, ``source_descriptor_datatype_yyyyMMdd_vNNN``. For example, ``source_datatype_descriptor_yyyyMMdd`` would result in filename convention like this, ``imap_l1a_norm-raw_mag_20241122``, whereas ``source_descriptor_datatype_yyyyMMdd_vNNN`` would result in filename like this, ``imap_mag_l1a_norm-raw_20241122_v001``. See <https://imap-processing--1179.org.readthedocs.build/en/1179/development-guide/style-guide/naming-conventions.html#> for more details
```

Reply:
I'm happy with this! I'm still looking for a bit of clarification as to whether anyone working on IMAP would expect anything except for the IMAP format?

Reply:
On IMAP, no. We all use that `source_descriptor_datatype_yyyyMMdd_vNNN` convention.